doc/build/dialects/postgresql.rst

```diff
@@ -154,6 +154,68 @@ For example:
       during=DateTimeRange(datetime(2013, 3, 23), None)
   )
 
+MultiRange Types
+~~~~~~~~~~~~~~~~
+
+The new MultiRange column types found in PostgreSQL 14 onwards are
+catered for by the following types:
+
+.. autoclass:: INT4MULTIRANGE
+
+
+.. autoclass:: INT8MULTIRANGE
+
+
+.. autoclass:: NUMMULTIRANGE
+
+
+.. autoclass:: DATEMULTIRANGE
+
+
+.. autoclass:: TSMULTIRANGE
+
+
+.. autoclass:: TSTZMULTIRANGE
+
+
+The types above get most of their functionality from the following
+mixin:
+
+.. autoclass:: sqlalchemy.dialects.postgresql.ranges.RangeOperators
+    :members:
+
+.. warning::
+
+  The multirange type DDL support should work with any PostgreSQL DBAPI
+  driver, however the data types returned may vary.
+
+When instantiating models that use these column types, you should pass
+whatever data type is expected by the DBAPI driver you're using for
+the column type.
+
+For example:
+
+.. code-block:: python
+
+  from psycopg.types.range import Range
+  from pscyopg.types.multirange import Multirange
```

Comment:
**mike bayer** (zzzeek) wrote:

OK....lets at least add here that we are using the "psycopg" driver, so people have some idea what we're talking about

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3728

Reply:
updated